src/css/bs-chart.scss

```diff
@@ -14,23 +14,21 @@
   margin-left: 8px;
 }
 
-.bs-chart__group:first-child .bs-chart__card {
-  background: #39B54A;
-}
-
-.bs-chart__group:last-child .bs-chart__card {
-  background: #d0901a;
-}
-
 .bs-chart__card {
   width: 110px;
   flex-direction: column;
   align-items: center;
+  background: #39B54A;
   border-radius: 8px;
   color: #fff;
   padding: 8px;
   font-size: 11px;
   font-weight: 600;
+
+  &--deprecated {
+    @extend .bs-chart__card;
+    background: #d0901a;
+  }
```

Comment:
Styles don't get copied/propagated across versions. So when I removed the IE 11 bubble from the `docs/browser-support` page, that colored other bubble this fun mustard yellow instead.

If we end up rolling this (`docs/browser-support.md`) page into the support page, I'll roll back this change (and the changes to versioned browser-support pages)